js/settings.js

```diff
@@ -37,7 +37,7 @@ window.addEventListener("DOMContentLoaded", function () {
 		storageConfig = JSON.stringify(storageArray);
 		// reset temp array for next Save click.
 		storageArray = [];
-		$.post(OC.generateUrl("apps/files_external_beeswarm/settings/admin"), {
+		$.post(OC.generateUrl("apps/files_external_ethswarm/settings/admin"), {
```

Comment:
I think we should not use JQuery. @mwalbeck not sure here.

`$.post` would then be 
```
fetch("OC.generateUrl("apps/files_external_ethswarm/settings/admin", {method: 'post', body: { storageconfig: storageConfig }})
.then(...)
.catch(...)```

Reply:
setting.js is no longer used, it should be removed.  It was used in the 1st version of the app, using Jquery and not Vue